### ISSUE TITLE
Config option for hacks and issue with rm2 gestures

### ISSUE THREAD

**Issue by Timrich01** [mb1986/rm-hacks#37]
Hi, thanks for the great work!

One issue I noticed, was that the scrolling gesture with two fingers works significantly worse with the hacks installed (maybe because of the one finger slide down?).

And a feature request: Would it be possible to add a config option / toggle option for the hacks? So they can be configured or activated based on user preference.

**Comment by Eeems** [comment]
Duplicate of #33

**Comment by Timrich01** [comment]
> Duplicate of #33

updated it accoringly

**Comment by mb1986** [comment]
Is this issue with gestures related only to scroll ones or all of the gestures?

**Comment by Timrich01** [comment]
Scroll ones seem to be affected the most. Mainly in the center of the screen too, on the edges it works similar to normal. 
Zoom gestures slightly.

Tap gestures and other gestures (swipe from top with one/two fingers, swipe from bottom/ left/ right) are working as normal.

**Comment by mb1986** [comment]
Is it better when you disable the toolbar toggle gesture in the new rM Hacks settings of v0.0.5?

**Comment by Timrich01** [comment]
just updated and yes it seems to be connected to the toolbar toggle gesture. With it being deactivated, there is no issue scrolling or zooming.

**Comment by mb1986** [comment]
Thanks for the confirmation. What I do, I change the native up/down swipes to work only for two fingers, while one-finger swipe is used for toolbar toggling, and in the result native swipes work worse as you've mentioned.

I don't have any solution for now besides disabling toolbar toggle gesture. Let's keep this open.

Thinking out loud, we may have different zones/areas for different swipe gestures to be recognized.

**Comment by Timrich01** [comment]
Yeah, this seems to be the case. Maybe the parameters can be tweaked on the delay for two finger swipes vs two one-finger swipes. So even if the second finger is a bit delayed the gesture still counts as two finger swipe.

**Comment by Potjoe-97** [comment]
> Thanks for the confirmation. What I do, I change the native up/down swipes to work only for two fingers, while one-finger swipe is used for toolbar toggling, and in the result native swipes work worse as you've mentioned.

Actually in my opinion, this is a feature and not a bug : native swipes are a nightmare to deal with, because they triggered way too many times unexpectedly. The fact that native up/down requires two fingered instead of one is a welcomed improvement over original behaviour, since it is not triggered unexpectedly anymore, and facilitate user navigation and focus on page.

**Comment by atngames** [comment]
Rm had greatly improved the way their gestures work.

Do you still have issues with it?
